### DISCUSSION TITLE
How to replace the 'About Avalonia' dialog window with a custom dialog window

### DISCUSSION THREAD

**Discussion by yeaicc** [AvaloniaUI/Avalonia#16535] [Q&A] [answered]
How to replace the 'About Avalonia' dialog window with a custom dialog window

I have added some NativeMenu, but after running the application, there will always be a menu "Avalonia Application" appearing. I want to remove it

**Comment by thevortexcloud** [comment]
Why do you need to do this? Can't you just make a custom dialogue from scratch and then display that instead?

**Reply by yeaicc** [reply]
I have added some NativeMenu, but after running the application, there will always be a menu "Avalonia Application" appearing. I want to remove it

**Comment by yeaicc** [comment, marked as answer]
1. Open `App.axaml` and set the Application's name property to "Your app name"
   ``` xaml
   <Application ...
             Name="Your app name"
            ...>
   ```
2. Add the following code to this file:
   ``` xaml
   <Application ...
       ...


      <NativeMenu.Menu>
          <NativeMenu>
              <!-- Follow the desired settings -->
              <NativeMenuItem Header="_About"></NativeMenuItem>
              <NativeMenuItem Header="_Settings"></NativeMenuItem>
          </NativeMenu>
      </NativeMenu.Menu>

   </Application>

   ```
